st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -144,15 +153,18 @@ def get_all(self, prefix=None, scope=SYSTEM_SCOPE, user=None, decrypt=False, **k
         return kvp_apis
 
     @jsexpose(arg_types=[str, str, str], body_cls=KeyValuePairSetAPI)
-    def put(self, kvp, name, scope=SYSTEM_SCOPE):
+    def put(self, kvp, name, scope=None):
```

Comment:
Same here, let's please also default the argument value here. Or do you need to differentiate between system scope being a default value and a system scope requested by the user?